odfuzz/entities.py

```diff
@@ -692,9 +690,18 @@ def get_depending_data(self):
 class OrderbyQuery(QueryOption):
     """The search query option."""
 
-    def __init__(self, entity, restrictions):
+    def __init__(self, entity, restrictions, restrictions_group):
```

Comment:
Why is here new parameter, restriction_group, while in DirectBuilder the same data are provided just as substructure of restrictions? I would expected that nothing would change here as well.

Reply:
The first Parameter (restrictions) calls the forbidden_options method of the class RestrictionGroup which does the not provide the Exclusion_List we require to restrict the Properties. This is just a subpart of the RestrictionsGroup.

It provides us with the YAML file restriction Object:
Exclude:
    $filter:
        Categories:
            - Description
            - Picture
        Employees:
            - Photo
            - HireDate
        $F_ALL$:
            - replace
    $orderby:
        Categories:
            - Picture
            - Description
       
What we actually need is the restrictions passed as a json object to the RestrictionGroup Class when we initialize the RestrictionGroup class.

The second Parameter (restrictions_group) calls the excluded_options method which provides us with the ExclusionList Json Object to actually restrict the properties.

This Method provides us with the actual Json Object which we need to restrict the properties:
exclusion_dict = {"$ENTITY_SET$":{".*": {"Properties":["ProductID"],"Nav_Properties":[]}}}

This is what we require to restrict the properties and not the previous object.